runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/BeamFnMapTaskExecutorFactory.java

```diff
@@ -480,6 +548,29 @@ public Node typedApply(RegisterRequestNode input) {
     return ptransformIdToSideInputIdToPCollectionViewBuilder.build();
   }
 
+  /**
+   * Returns a table where the row key is the PTransform id, the column key is the side input id,
+   * and the value is the corresponding PCollectionView.
+   */
+  private static Map<RunnerApi.ExecutableStagePayload.SideInputId, PCollectionView<?>>
```

Comment:
trying to understand why it is called table, essentially it has map of key being a Tuple(TransformId, SideInputId), is that correct?